documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -175,7 +178,7 @@ EOF
 ----
 <1> Allowed values are `pod` and `multus`.
 <2> You can use either the `id` _or_ the `name` parameter to specify the source network.
-<3> Specify the VMware network MOR, the {rhv-short} network UUID, or the {osp} network UUID.
+<3> Specify the VMware network MOR, the {rhv-short} network UUID, the {osp} network UUID, or the OVA network UUID.
```

Comment:
@ahadas its not directly connected to this PR but it got me thinking, the network UUID is something we generate automatically in the backed, so for the users to have this value they will need to query the inventory or the OVA server. Maybe it be more logical to use the network name instead? (also its a minor change so I can upload it now and we can also match the documentation in the same breath to avoid future changes)

Reply:
yeah, I also thought about this point - that the UUID are not values that we can fetch from the source environment but I don't think that's an issue because when it's done programmatically, it doesn't make sense that the code reaches out to the source environment but rather query what we have in the inventory and then create the migration resources based on that information
support specifying network names can be useful regardless of the above

Reply:
@bkhizgiy @ahadas  Not sure I understand -- should I change <3> to "Specify the network name of the source provider" -- and that is all it will say?

Reply:
@RichardHoch Not yet, I was referring to our implementation in the backend, simply seeing it in the documentation got me thinking about it :).
We can add it later once I post the implementation additiion.